digid_eherkenning/views/eherkenning.py

```diff
@@ -110,3 +112,31 @@ def get(self, request):
         auth.login(request, user)
 
         return HttpResponseRedirect(self.get_success_url())
+
+
+def get_xml_eherkenning_metadata(request):
+
+    eherkenning_config = EherkenningMetadataConfiguration.get_solo()
+    try:
+        eherkenning_config.clean()
+    except ValidationError:
+        return HttpResponseBadRequest(
+            _("Something went wrong during metadata recovery. Please, contact ....")
```

Comment:
Todo?

Reply:
refactored into a re-usable class based view which has the appropriate error